docs/frontends/dahlia.md

```diff
@@ -11,7 +11,7 @@ Then, clone the repository and build the Dahlia compiler:
 ```
 git clone https://github.com/cucapra/dahlia.git
 cd dahlia
-sbt install
+sbt compile
 sbt assembly
```

Comment:
Just checking: does it work if we _just_ do `sbt assembly`? Seems like a preceding `sbt compile` should probably not be necessary; `sbt assembly` would compile everything before building the archive.

Reply:
I think assembly depends on compile and therefore should run it but I’m paranoid about my sbt skills so leaving both in is fine IMO